client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -331,10 +335,34 @@ public void testCloseNonExistentIndex() throws IOException {
 
         CloseIndexRequest closeIndexRequest = new CloseIndexRequest(nonExistentIndex);
         ElasticsearchException exception = expectThrows(ElasticsearchException.class,
-                () -> execute(closeIndexRequest, highLevelClient().indices()::close, highLevelClient().indices()::closeAsync));
+            () -> execute(closeIndexRequest, highLevelClient().indices()::close, highLevelClient().indices()::closeAsync));
         assertEquals(RestStatus.NOT_FOUND, exception.status());
     }
 
+    public void testRefresh() throws IOException {
+        // 10 shards per index by default (5 primary shards + 5 replica shards)
+        final int DEFAULT_SHRADS_NUMBER_PER_INDEX = 10;
+        {
+            String index = "index";
+            createIndex(index);
```

Comment:
you can also specify the number of shards yourself so you don't rely on any default.